org.kde.PlatformInputContexts.MaliitSailfishOS.yaml

```diff
@@ -0,0 +1,30 @@
+id: org.kde.PlatformInputContexts.MaliitSailfishOS
+branch: 5.14
+runtime: org.kde.Platform
+build-extension: true
+sdk: org.kde.Sdk
+runtime-version: 5.14
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: maliit-framework-plugin
+    buildsystem: simple
+    build-commands:
+      - cmake -Denable-docs=OFF -Denable-tests=OFF -Denable-glib=off -Denable-xcb=ON -Denable-wayland=OFF -Denable-wayland-gtk=OFF -Denable-qt5-inputcontext=ON -Denable-hwkeyboard=OFF
+      - make -j4 maliitplatforminputcontextplugin
+      - install -D -t ${FLATPAK_DEST}/platforminputcontexts libmaliitplatforminputcontextplugin.so
+    sources:
+      - type: git
+        url: https://github.com/sailfishos-flatpak/maliit-framework.git
+        branch: flatpak
+        commit: c1ce5ed850597cf7b0a1ea22cd6450bf4217092e
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/appdata
+      - cp org.kde.PlatformInputContexts.MaliitSailfishOS.appdata.xml ${FLATPAK_DEST}/share/appdata
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Can we skip appdata then?

Reply:
Can we skip appdata then?

Reply:
As there is a silence regarding skipping appdata - any example regarding it? Do I just rename it and mv to $PREFIX/share/metainfo/%{id}.metainfo.xml?